diff -up xtables-addons-kmod-1.39/xtables-addons-1.39/extensions/ipset-6/ip_set_core.c.fakek xtables-addons-kmod-1.39/xtables-addons-1.39/extensions/ipset-6/ip_set_core.c
--- xtables-addons-kmod-1.39/xtables-addons-1.39/extensions/ipset-6/ip_set_core.c.fakek	2011-11-23 21:20:09.628117659 +0100
+++ xtables-addons-kmod-1.39/xtables-addons-1.39/extensions/ipset-6/ip_set_core.c	2011-11-23 21:20:12.963316519 +0100
@@ -1170,7 +1170,7 @@ ip_set_dump(struct sk_buff *skb, struct 
 	genl_unlock();
 	ret = netlink_dump_start(ctnl, skb, nlh,
 				  ip_set_dump_start,
-#if LINUX_VERSION_CODE < KERNEL_VERSION(3, 1, 0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 41)
 				  ip_set_dump_done);
 #else
 				  ip_set_dump_done, 0);
